trello-solarized-dark.user.css

```diff
@@ -1,3 +1,14 @@
+/* ==UserStyle==
+@name         Trello Solarized Dark Grey
+@version      0.6.0
+@description  A Dark Travis-CI style with purple highlights
+@namespace    https://github.com/ristomatti/trello-solarized-dark
+@author       Ristomatti Airo
+@homepageURL  https://github.com/ristomatti/trello-solarized-dark
+@supportURL   https://github.com/ristomatti/trello-solarized-dark/issues
+@updateURL    https://raw.githubusercontent.com/ristomatti/trello-solarized-dark/master/trello-solarized-dark.user.css
+==/UserStyle== */
+
 @-moz-document url("https://trello.com/"), regexp("^https?://trello.com/[cb]{1}.*$") {
 /**
 * Trello Solarized Dark Grey v0.6 (2018-08-28)
```

Comment:
Please remove version number from this line if it'll anyway be in the new header section.

Reply:
I don't know what to do with the date then though. I guess it can be removed also. It's useless info to anyone else but myself anyway.

Reply:
Added an update to remove both.

 In theory its probably useful other to know the date in case they want to check versions. Theoretically this should be handled by the metadata section though so perhaps that handles that. Traceability of versiosn to dates may be then handled by github if that helps?